.redhat-ci.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# F25 currently has 2.2.1, so install from pypi
+pip install ansible==2.2.2.0
```

Comment:
Ahh good catch. Didn't notice the `requirements.txt` in the repo. Will open a PR to change that.